components/stylesheets/Header.module.css

```
@@ -5,6 +5,7 @@
     width: 100%;
     padding: 15px 30px 15px 30px;
     box-sizing: border-box;
+    margin-top:17px;
```

Vraiment pas un deal breaker:
```suggestion
    margin-top: 17px;
```